rs/config/src/collection.rs

```diff
@@ -127,6 +127,9 @@ pub struct CollectionConfig {
     /// Default: 0 (not using max time to flush)
     #[serde(default = "default_max_time_to_flush_ms")]
     pub max_time_to_flush_ms: u64,
+
+    pub topic_prefix: String,
+    pub group_id: String,
```

Comment:
use `#[serde(default = ...)]`

Reply:
Also, you need to handle the case where it doesn't want to use Kafka. Let's by default disable kafka, and only enable if they set `use_distributed_log_as_wal : true`